integration/network/network_test.go

```diff
@@ -17,6 +18,8 @@ import (
 	is "gotest.tools/v3/assert/cmp"
 	"gotest.tools/v3/icmd"
 	"gotest.tools/v3/skip"
+	"github.com/docker/docker/integration/internal/swarm"
+	"fmt"
```

Comment:
`fmt` should be grouped with the rest of the stdlib packages above.
Maybe run `goimports`?

Reply:
Thanks for your advice!
I have revised my commit according to your suggestion!

ping @cpuguy83

Reply:
I have merged two commits into one.
Please take a look at it!